combined_browser_metrics/views/cohort_daily_statistics.view.lkml

```diff
@@ -4,12 +4,58 @@ view: +cohort_daily_statistics {
 
   dimension: days_retained {
     type: number
-    sql: DATE_DIFF(${cohort_date}, ${activity_date}, DAY) ;;
+    sql: DATE_DIFF(${activity_date}, ${cohort_date}, DAY) ;;
   }
 
-  measure: retention {
-    sql: SAFE_DIVIDE(SUM(${num_clients_active_on_day}), SUM(${num_clients_in_cohort})) ;;
-    value_format: "0.00%"
+  dimension: normalized_app_name {
+    label: "Browser Name"
+    sql: ${TABLE}.normalized_app_name ;;
+    type: string
+  }
+
+  dimension: normalized_channel {
+    label: "Channel"
+    sql: ${TABLE}.normalized_channel ;;
+    type: string
+  }
+
+  dimension: normalized_os {
+    label: "OS"
+    sql: ${TABLE}.normalized_os ;;
+    type: string
+  }
+
+  dimension: normalized_os_version {
+    label: "OS Version"
+    sql: ${TABLE}.normalized_os_version ;;
+    type: string
+  }
+
+  dimension: attribution_source {
+    sql: ${TABLE}.attribution_source ;;
```

Comment:
Might want to give an explicit `group_item_label: "Attribution Source" or let Looker convention?